src/containers/JoinUs.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import CInput from '../components/Cinput';
+import CButton from '../components/Cbutton';
+import { ExternalPages } from '../constants/externalPages';
+
+import fetch from '../utils/request';
+
+import JoinDancing from 'public/images/joinDancing.png';
+
+const JoinUs = () => {
+  const [value, setValue] = useState('');
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value);
+  };
```

Comment:
no need for this. Just give the CInput a `name` attribute and you will be able to access its value inside the `handleSubmit` using e.target.NAME.value